tests/virt_autotest/host_upgrade_step2_run.pm

```diff
@@ -39,6 +41,18 @@ sub run {
     if (($upgrade_release >= 15) && ($installed_release ne $upgrade_release)) {
         upload_logs('/root/autoupg.xml');
     }
+
+    #get the version that the host is installed to
+    my $host_installed_version = get_var('VERSION_TO_INSTALL', get_var('VERSION', ''));    #format 15 or 15-SP1
```

Comment:
```suggestion
    my $host_installed_version = get_var('VERSION_TO_INSTALL', get_var('VERSION'));    #format 15 or 15-SP1
```

I think in case of `VERSION` variable no point for default value because it is impossible to run openQA test without this variable defined

Reply:
Are you sure ?  If yes, I am happy to make the change. Also the old implementation in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/virt_autotest/host_upgrade_step2_run.pm Line35

Reply:
Is there theoretical reason to run the job with 'VERSION' undefined ? Yes there is. For example you can clone the job and reset VERSION variable. But usually jobs triggered with `/usr/bin/openqa-client isos post` and openqa-client deny triggering jobs if you try to call him without VERSION variable. But actually it was just kind advise defaulting here not bring any harm if it makes you feel safe let's keep it :)